rosbridge_library/src/rosbridge_library/internal/message_conversion.py

```diff
@@ -61,8 +61,6 @@
         "uint32",
         "int64",
         "uint64",
-        "float32",
-        "float64",
```

Comment:
https://github.com/RobotWebTools/rosbridge_suite/pull/692#issuecomment-1008881740

Since I changed to use `type_map["int"]`, I removed them.
https://github.com/RobotWebTools/rosbridge_suite/pull/692/files#diff-212be7c9fa67fbb1f8af9069876e393724e387108a685d2cc0a2f2375b561ddbR253-R254